src/applications/discharge-wizard/constants/display-conditions.js

```diff
@@ -0,0 +1,63 @@
+import { RESPONSES } from './question-data-map';
+
+const get15YearsPast = () => `${new Date().getFullYear() - 15}`;
+
+const {
+  REASON_1,
+  REASON_2,
+  REASON_3,
+  REASON_4,
+  REASON_5,
+  //   REASON_8,
+  REASON_6,
+  REASON_7,
+  //   INTENTION_1,
+  //   INTENTION_2,
+  //   COURT_MARTIAL_1,
+  //   COURT_MARTIAL_2,
+  //   COURT_MARTIAL_3,
+  //   DISCHARGE_TYPE_1,
+  //   DISCHARGE_TYPE_2,
+} = RESPONSES;
+
+export const DISPLAY_CONDITIONS = Object.freeze({
+  SERVICE_BRANCH: {},
+  DISCHARGE_YEAR: {
+    SERVICE_BRANCH: [],
```

Comment:
I can appreciate the simplified approach here, but given these display conditions, Discharge Year doesn't require a response for the Service Branch question in order to display (though I understand there might be other logic allowing it to display correctly). I think, if we're going to use display conditions at all, they need to be verbose (include conditions for every question preceding). This allows us to insert or remove questions later down the line without having to refactor things.

Reply:
Thanks Randi for the insight. That makes sense. I went ahead and updated the DCs to include service branch and discharge year again